steering_controllers/ackermann_steering_controller/test/test_ackermann_steering_controller.cpp

```diff
@@ -0,0 +1,566 @@
+// Copyright (c) 2022, Stogl Robotics Consulting UG (haftungsbeschränkt) (template)
```

Comment:
```suggestion
// Copyright (c) 2022, Stogl Robotics Consulting UG (haftungsbeschränkt)
```

Didn't check the rest or the file for now.

Reply:
I have removed the tests, for now, and will work on them later when we have a consensus on the project structure.